lessons/04-client-jquery-ajax/HOMEWORK.md

```diff
@@ -0,0 +1,11 @@
+### Class 4 Homework
+For the homework, you will be integrating mongoDB into your Cat App from last class using mongoose. Now you can store the cats you create forever, and create new features that allow you to sort those cats in various ways.
+
+For this homework, you will need to add at least one feature that integrates with your database and uses an advanced query. For example, you could only show cats in a certain age range, or display cats by date created.
+
+**Challenging** Visualize your cat collection using D3 or your favorite data visualization library.
```

Comment:
Prior to jQuery, or really any major introduction to client side coding, this may be a bit too challenging (and off-topic?) of a challenge. Perhaps interested students could try out a more complicated aspect of mongoose (explore embedding vs. referencing?), implement a remote db on mongolab, or repeat the Cat App DB integration with mysql?